### ISSUE TITLE
NPM package codemirror@5.14.0 is missing the LICENSE file

### ISSUE THREAD

**Issue by Dominator008** [codemirror/CodeMirror#3980]
Repro:
```sh
npm install codemirror@5.14.0
cd node_modules/codemirror/ && ls LICENSE
```
Output:
```
ls: cannot access LICENSE: No such file or directory
```
The same situation happened with 5.13.2 (https://github.com/codemirror/CodeMirror/issues/3940).
Please do a patch release after fixing this and merging #3976, #3977 and #3978. Thanks!

**Comment by marijnh** [comment]
What the hell. This time I am entirely sure that the file was present when I did `npm publish`, and it's not listed in `.npmignore`. There don't seem to be any other files missing from the tarball.

I'd like to get to the bottom of this, rather than continuing to have arbitrary misfiring releases.

**Comment by Dominator008** [comment]
Not sure if you hit https://github.com/npm/npm/issues/5082

**Comment by marijnh** [comment]
Yes, I think that's the same bug. And what an awful bug it is. Okay, I'll shoot out a minor release (and hope it doesn't happen again).

**Comment by Dominator008** [comment]
Anecdotal [comment](https://github.com/npm/npm/issues/5082#issuecomment-101780285 ) seems to suggest that using an explicit `files` section in `package.json` helped: 
https://github.com/lifegadget/ui-responsive/commit/38c8c267045d15a699626bb7922a55c30f420a19.

A workaround might be doing a `npm pack` before the `npm publish` and make sure everything is in the tarball.

**Comment by marijnh** [comment]
Released 5.14.2, which seems to have a LICENSE file

**Comment by Dominator008** [comment]
Thanks!
